src/SegmentedControl/SegmentedControl.test.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import {render} from '@testing-library/react'
+import {EyeIcon, FileCodeIcon, PeopleIcon} from '@primer/octicons-react'
+import userEvent from '@testing-library/user-event'
+import {behavesAsComponent, checkExports, checkStoriesForAxeViolations} from '../utils/testing'
+import {SegmentedControl} from '.' // TODO: update import when we move this to the global index
+
+const segmentData = [
+  {label: 'Preview', iconLabel: 'EyeIcon', icon: () => <EyeIcon aria-label="EyeIcon" />},
```

Comment:
Just curious, is there a reason you're setting `aria-label` on the SVG and the button? I think by default the SVGs rendered by octicons are aria-hidden